.gitignore

```diff
@@ -0,0 +1,4 @@
+/.idea
+/target
+/*.lock
```

Comment:
Cargo.lock should probably not be ignored. It is also already included in the repository. https://doc.rust-lang.org/cargo/faq.html#why-do-binaries-have-cargolock-in-version-control-but-not-libraries

Reply:
I removed the /*.lock from the file :)